app/custom_domain_validation.py

```diff
@@ -0,0 +1,33 @@
+from app.db import Session
+from app.dns_utils import get_cname_record
+from app.models import CustomDomain
+
+
+class CustomDomainValidation:
+    def __init__(self, dkim_domain: str):
+        self.dkim_domain = dkim_domain
+
+    def get_dkim_records(self, custom_domain: CustomDomain) -> {str: str}:
```

Comment:
this method seems to always return the same result, should we create a constant for it instead?

Reply:
I'll move it to be generated on instantiation.

Reply:
Done